Models/Config.cs

```diff
@@ -0,0 +1,126 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Miki.Logging;
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Threading.Tasks;
+
+namespace Miki.Bot.Models
+{
+    [Table("Configuration")]
+    public class Config
+    {
+        [Key]
+        [Column("Id")]
+        public Guid Id { get; set; } = new Guid();
```

Comment:
This gets cached in Postgres as a guid of value 'new Guid()'. Would recommend to do it in a constructor instead